### ISSUE TITLE
virtualenv cause anaconda server not functioning

### ISSUE THREAD

**Issue by casouri** [syl20bnr/spacemacs#9720]
#### Description :octocat:
entering python virtualenv cause error on anaconda server.
I have tried reinstalling Jedi and running `pip install --upgrade "jedi>=0.9.0" "json-rpc>=1.8.1" "service_factory>=0.1.5"`.

#### Reproduction guide :beetle:
- create a virtualenv with `virtualenv somepath/env`
- Start Emacs
- goes into python-mode
- SPC m V a
- activate somepath/env
- wait for a few seconds
- error pops up

*Observed behaviour:* :eyes: :broken_heart:
anaconda not functioning

*Expected behaviour:* :heart: :smile:
anaconda functions

#### System Info :computer:
- OS: darwin
- Emacs: 25.3.1
- Spacemacs: 0.200.9
- Spacemacs branch: develop (rev. 9ce0310)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(casouri ;; my own layer contains nyan-mode
 (python :variables python-enable-yapf-format-on-save t)
 (c-c   :variables c-c  -enable-clang-support t)
 helm
 (auto-completion :variables auto-completion-enable-snippets-in-popup t)
 better-defaults emacs-lisp git markdown org
 (shell :variables shell-default-height 40 shell-default-position 'bottom)
 spell-checking syntax-checking version-control themes-megapack sphinx)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

#### Backtrace :paw_prints:
```
Traceback (most recent call last):
  File "<string>", line 4, in <module>
  File "/Users/yuan/.emacs.d/.cache/anaconda-mode/0.1.8/anaconda_mode-0.1.8-py3.6.egg/anaconda_mode.py", line 17, in <module>
    from jedi import Script, NotFoundError
ImportError: cannot import name 'NotFoundError'
Traceback (most recent call last):
  File "<string>", line 4, in <module>
  File "/Users/yuan/.emacs.d/.cache/anaconda-mode/0.1.8/anaconda_mode-0.1.8-py3.6.egg/anaconda_mode.py", line 17, in <module>
    from jedi import Script, NotFoundError
ImportError: cannot import name 'NotFoundError'
```

**Comment by et2010** [comment]
It seems you're still using anaconda_mode 0.1.8. You can update to the latest version by deleting the directory `~/.emacs.d/.cache/anacond-mode`

**Comment by casouri** [comment]
updating spaceman’s solved it. Thank you!

**Comment by vorstrelok** [comment]
I had exact same issue, and the same backtrace.
I beleive that root cause is that NotFoundError was removed in recent version of jedi (0.11 or later which you probably use) but you still use older version of anaconda-mode (like v0.1.8).

I was able to fix this simply by upgrading all my packages (and hence anaconda-mode) via update packages button in the start buffer.

all kudos to @freyes in this issue https://github.com/proofit404/anaconda-mode/issues/225
